imgui_node_editor.cpp

```diff
@@ -1123,9 +1123,10 @@ void ed::EditorContext::Begin(const char* id, const ImVec2& size)
 
     if (!m_IsInitialized)
     {
-        // Cycle canvas so it has a change to setup its size before settings are loaded
-        m_Canvas.Begin(id, canvasSize);
-        m_Canvas.End();
+        // Cycle canvas, so it has a chance to set up its size before settings are loaded
+        bool success = m_Canvas.Begin(id, canvasSize);
+        if (success)
+            m_Canvas.End();
```

Comment:
I was about to create a similar PR and saw your PR :)

I'd suggest you change this to

```cpp
        if (m_Canvas.Begin(id, canvasSize))
            m_Canvas.End();
```

Too keep in line with the rest of the codebase :)

Reply:
Hi, I did the change